source/JsMaterialX/README.md

```diff
@@ -74,21 +78,20 @@ cmake --build .
 
 
 ### Output
-After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in `./<build_folder>/source/JsMaterialX/`.
+After building the project the `JsMaterialX.wasm` and `JsMaterialX.js` files can be found in the global install directory of this project.
 
 ### Install
-To install the results into the install directory run
+To install the results into the test directory run
 ```sh
 cmake --build --target install
 ```
 from the build directory.
 
-### Testing
-The JavaScript tests are located in `<root_dir>/source/JsMaterialX/test` folder and are defined with the `.spec.js` suffix.
-Most of these tests are the same as the Python [main.py tests](../../python/MaterialXTest/main.py).
+## Testing
+The JavaScript tests are located in the `test` folder and are defined with the `.spec.js` suffix.
 
 #### Setup
-These tests require `node.js`. This is a part of the emscripten environment. So make sure to call `emsdk_env` before running the steps described below.
+These tests require `node.js`, which is shipped with the emscripten environment. Make sure to execute the `emsdk/emsdk_env` script before running the steps described below.
```

Comment:
hum, not sure if it changes on windows but on mac I dont execute it but run `source emsdk/emsdk_env`

Reply:
good catch